charts/thoras/values.yaml

```diff
@@ -60,6 +60,19 @@ thorasApiServer:
   port: 443
   logLevel: "debug"
 
+thorasApiServerV2:
+  enabled: false
+  containerPort: 8443
```

Comment:
Question -- will we be mounting a TLS key pair into the pod and doing our own TLS termination in code?

Reply:
Its mounted in:
https://github.com/thoras-ai/helm-charts/blob/d7476ed9c041b873f70c07c7f9782b79af97faf2/charts/thoras/templates/api-server-v2/deployment.yaml#L32